monkestation/code/modules/martial_arts/granters/tribal_claw_gr.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/book/granter/martial/tribal_claw
+	martial = /datum/martial_art/tribal_claw
+	name = "old scroll"
+	martial_name = "tribal claw"
+	desc = "A scroll filled with ancient draconic markings."
+	greet = "<span class='sciradio'>You have learned the ancient martial art of the Tribal Claw! You are now able to use your tail and claws in a fight much better than before. \
```

Comment:
please replace all the spans in messages with the helper macros

Reply:
problematic items replaced